Vagrantfile

```diff
@@ -200,6 +200,16 @@ Vagrant.configure(2) do |config|
     c.vm.synced_folder ".", "/vagrant", type: "virtualbox"
   end
 
+  # Windows Server 2016
```

Comment:
I assume this is not directly related to this PR?

Reply:
It's only related in that I was testing these build changes on win2016 and we didn't have a machine for it yet.